test/Constraints/closures.swift

```diff
@@ -202,7 +202,7 @@ func acceptNothingToInt (_: () -> Int) {}
 func testAcceptNothingToInt(ac1: @autoclosure () -> Int) {
   // expected-note@-1{{parameter 'ac1' is implicitly non-escaping because it was declared @autoclosure}}
   acceptNothingToInt({ac1($0)})
-  // expected-error@-1{{cannot convert value of type '(_) -> Int' to expected argument type '() -> Int'}}
+  // expected-error@-1{{contextual closure type '() -> Int' expects 0 arguments, but 1 were used in closure body}}
```

Comment:
This diagnostic is from `closure_argument_list_tuple` where I didn't touch (only use it). What I added was `closure_argument_list_single_tuple`. I think it is a minor bug sticking with `closure_argument_list_tuple` where it didn't use `%select` for `{was|were}`. Do you think the best way would be to put it in this commit? or have a separate bug fixing commit instead?